components/ibm-components/watson/deploy/component.yaml

```diff
@@ -14,7 +14,7 @@ name: 'Deploy Model - Watson Machine Learning'
 description: |
   Deploy stored model on Watson Machine Learning as a web service.
 metadata:
-  labels: {platform: 'IBM Watson Machine Learning'}
+  labels: {platform: 'IBM-Watson-Machine-Learning'}
 inputs:
```

Comment:
Let`s see if  `annotations` can work, then we retain the original names

Reply:
I moved the labels to annotations since annotations allow white spaces